mautic/mautic
# Connectwise campaign groups

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | Y
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
This PR adds campaign groups to Integration Campaigns filter for segments.
[//]: # ( As applicable: )
#### Steps to test this PR:
1. Apply this PR and you should be able to create segments from Connectwise campaign groups.
2. your connectwise plugin must be enabled and authenticated.
3. go to segments select the 'Integrations Campaigns' filter and in the dropdown you will see campaign groups from Connectwise if there are any. Contacts in that group should be imported from Connectwise to Mautic and a segment should be created.
